src/Components/test/testassets/Components.TestServer/RazorComponents/Shared/EnhancedNavLayout.razor

```diff
@@ -25,4 +25,6 @@
     <NavLink href="nav/location-changed/server-and-wasm">LocationChanged/LocationChanging event (server-and-wasm)</NavLink>
 </nav>
 <hr/>
-@Body
+<main>
+    @Body
+</main>
```

Comment:
This change is needed to work around a limitation in the DOM diffing (can't merge two elements of which only one is a LogicalElement). Doesn't affect the validity of the scenario.